src/components/AnonymousChat/child/ChannelPostNotificationMessage.tsx

```diff
@@ -1,47 +1,47 @@
 import * as React from 'react';
 import {StyleSheet, Text} from 'react-native';
 
-import useProfileHook from '../../../hooks/core/profile/useProfileHook';
 import {BaseChannelItemTypeProps} from '../../../../types/component/AnonymousChat/BaseChannelItem.types';
 import {colors} from '../../../utils/colors';
-import {fonts} from '../../../utils/fonts';
+import {fonts, normalize} from '../../../utils/fonts';
 
 const ChannelPostNotificationMessage = ({
   type = BaseChannelItemTypeProps.ANON_PM,
-  commenterId,
   commenterName,
   message
 }) => {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `type`, `commenterName`, and `message` props are not validated. It's a good practice to validate props using PropTypes or TypeScript interfaces for better error handling and debugging.
```diff
+ import PropTypes from 'prop-types';
  
  const ChannelPostNotificationMessage = ({
    type = BaseChannelItemTypeProps.ANON_PM,
    commenterName,
    message
  }) => {
...
+ };
+
+ ChannelPostNotificationMessage.propTypes = {
+   type: PropTypes.string,
+   commenterName: PropTypes.string,
+   message: PropTypes.string
+ };
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->